src/utils/subgraph.ts

```diff
@@ -298,3 +316,26 @@ export async function getAssetsBestPrices(
 
   return assetsWithPrice
 }
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
if TypeScript complains about missing return type definition then it is right. So instead of disabling the rule, we should actually follow it. From what I'm seeing this method returns `Promise<string>` so the next line should be:

```ts
export async function getHighestLiquidityDIDs(): Promise<string>
```

Reply:
Resolved this and the one mentioned before. My concern related to this implementation is that the assets displayed aren't the correct ones. 

On live we got:
![Screenshot from 2021-05-19 12-08-04](https://user-images.githubusercontent.com/49017601/118786980-06e45100-b89b-11eb-8206-c227d2de6597.png)

But on this PR we got:
![Screenshot from 2021-05-19 12-07-12](https://user-images.githubusercontent.com/49017601/118787028-16639a00-b89b-11eb-8933-7c6b12ffc278.png)

I'm on this right now

Reply:
mmmh, as posted in my last comment further down, I'm on the preview link and get what's in my screenshot, so that screenshot here shows different ones again. Are your latest changes pushed?

Reply:
pretty sure we do not render the order we want because we send a search query to aquarius and its results will be ordered by its default sort order, unless we do something. E.g. we need to order it based on the order we have in the DID list somehow

Reply:
Saw this with some delay, sorry. The screenshots were made before changes were made. I pushed them seconds ago.

Reply:
> pretty sure we do not render the order we want because we send a search query to aquarius and its results will be ordered by its default sort order, unless we do something. E.g. we need to order it based on the order we have in the DID list somehow

yes, and what I missed is that I don't exclude the assets in purgatory in the aquarius query